alembic/versions/2025_04_30_8e0b9e03054d_add_entries_for_raredisease_in_order_.py

```diff
@@ -6,10 +6,12 @@
 
 """
 
+import sqlalchemy
 import sqlalchemy as sa
```

Comment:
I can't seem to find the reason why we import this module twice.

Reply:
fixed